src/bindings/rust/README.md

```diff
@@ -0,0 +1,59 @@
+- infos =
+- infos/author = Philipp Gackstatter <philipp.gackstatter@student.tuwien.ac.at>
+- infos/status = experimental maintained
+- infos/provides =
+- infos/description =
+
+# Rust Bindings
+
+Rust bindings for libelektra.
+
+## Generation
+
+Using the `buildelektra-stretch-full` docker image, we have to install cargo and rustc via [rustup](https://rustup.rs).
+We also need to install rusts formattings tools with `rustup component add rustfmt`.
+
+Bindings are generated when buildling the `elektra-sys` crate using `rust-bindgen`. The `build.rs` script in the `elektra-sys` crate calls and configures bindgen. It also emits additional configuration for `rustc` to tell it what library to link against, and where to find it.
+Bindgen expects a `wrapper.h` file that includes all headers that bindings should be generated for. Since these headers could have additional includes, the `build.rs` also passes the elektra source and binary directory to clang, such that these includes can be found.
```

Comment:
This is already implementation detail? First explain the user-view of the Rust binding (How to use it.)